src/templates/kit/screens/FundraisingScreen.js

```diff
@@ -0,0 +1,607 @@
+import React, { useCallback, useMemo, useReducer, useState } from 'react'
+import { Decimal } from 'decimal.js'
+import PropTypes from 'prop-types'
+import { Field, GU, Help, Info, Tabs, TextInput, useTheme } from '@aragon/ui'
+import {
+  Header,
+  KnownAppBadge,
+  Navigation,
+  PercentageField,
+  ScreenPropsType,
+} from '..'
+
+const TABS = ['Overview', 'Advanced']
+const INPUT_SMALL = 17 * GU
+const INPUT_MEDIUM = 24 * GU
+
+const DEFAULT_VALUES = {
+  cliffPeriod: 90,
+  expectedGrowth: 200,
+  fundingPeriod: 14,
+  initialPricePerShare: 1.1,
+  initialPricePerShareInput: '1.1',
+  targetGoal: 25000,
+  presalePrice: 1,
+  presalePriceInput: '1',
+  tokensOffered: 90,
+  projectFunding: 10,
+  vestingSchedule: 365,
+  batchLength: 1,
+  slippageDai: 25,
+  slippageAnt: 100,
+  tapRate: 2500,
+  tapFloor: 2500,
+  maximumMonthlyUpdates: 50,
+}
+
+function BN(value) {
+  return new Decimal(value)
+}
+
+function labelDays(value) {
+  return value === 1 ? 'Day' : 'Days'
+}
+
+function labelBatch(value) {
+  return value === 1 ? 'Block' : 'Blocks'
+}
+
+function def(value, defaultValue) {
+  return value === undefined ? defaultValue : value
+}
+
+function intDef(value, defaultValue = 0) {
+  const intValue = parseInt(value, 10)
+  return isNaN(intValue) ? defaultValue : intValue
+}
+
+function floatDef(value, defaultValue = 0) {
+  const floatValue = parseFloat(value)
+  return isNaN(floatValue) ? defaultValue : floatValue
+}
+
+function dataDefaults(data) {
```

Comment:
mmm I don’t remember if there was an actual issue, but a difference I can see is that if a value on `data` is `undefined`, it would take precedence… but I don’t think there is any reason why we should have undefined values there. Let’s use your suggestion! :+1:

Reply:
Ah yes you're right on the defaulting. Let's not use my suggestion haha.